pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -411,9 +410,9 @@ func (ch *ClickHouseExportProcess) GetClickHouseConfig() ClickHouseConfig {
 }
 
 func ConnectClickHouse(config *ClickHouseConfig) (*sql.DB, error) {
-	_, addr, err := parseDatabaseURL(config.DatabaseURL)
+	proc, addr, tlsEnable, err := parseDatabaseURL(config.DatabaseURL)
```

Comment:
Is proc a commonly used abbreviation for protocol? Consider using proto?

Reply:
Updated. Thanks.